skp_generator/build.sh

```diff
@@ -0,0 +1,66 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Output each command, and quit immediately if any fail.
+set -ex
+
+# This function deletes everything except the skps directory and
+# what's already checked into git. This includes, for example, the
+# output of `flutter create`, any locally-checked-out flutter in this
+# directory, and the `build` directory (all of these get created by
+# subsequent action in this script).
+#
+# This function is called immediately on startup below, as well as
+# automatically when the script exits (see the "trap" below).
+function cleanup {
+    git clean -xffdq -e skps
+}
+
+cleanup
+#trap cleanup EXIT # COMMENT OUT THIS LINE TO LEAVE DEVELOPMENT ARTIFACTS (see README)
```

Comment:
nit: rather than leaving ac ommented out line, should we just have an argument to the script determine this?

Like `sh build.sh --no-clean` or something?

Reply:
i did not mean to leave this commented out oops

Reply:
Added support for `--no-clean` and updated README.